lib/user_agent/browsers/pod_2_watch.rb

```diff
@@ -0,0 +1,81 @@
+# frozen_string_literal: true
+
+class UserAgent
+  module Browsers
+    ##
+    # This class detects and handles the Pod2Watch browser on Apple Watch
+    class Pod2Watch < Base
+      # The Pod2Watch browser
+      BROWSER = 'Pod2Watch'
+
+      # The darwin product (providing a kernel version)
+      DARWIN = 'Darwin'
+
+      # The Operating System
+      WATCH_OS = 'watchOS'
```

Comment:
This is redundant now that you amended Base

Reply:
Removed DARWIN + WATCH_OS + PLATFORM constants.